.github/workflows/ci.yml

```diff
@@ -52,7 +52,6 @@ jobs:
               - 'karma.conf.js'
               - 'package.json'
             types:
-              - 'types/**'
```

Comment:
nit: After this, type changes are detected as `src` and thus full build/coverrage will be executed in CI.